CODEOWNERS

```diff
@@ -550,6 +550,7 @@
 /include/net/coap*.h                      @rlubos
```

Comment:
Minor nit, the commit subject do not need to mention "subsys", so `net: Introduce a network packet filter framework` is enough here.